src/types/models/index.ts

```diff
@@ -143,6 +144,18 @@ export interface MessageLead {
   }
 }
 
+export interface MessageLeadsListItem extends MessageLead {
```

Comment:
For listings, we call them `Listing` and `SearchListing`. Can we go with consistency?